.travis.yml

```diff
@@ -5,8 +5,9 @@ dist: bionic
 # https://devguide.python.org/#branchstatus
 python:
   - 3.6
-  - 3.7
```

Comment:
Why removing 3.7 and not 3.6. According to https://endoflife.date/python 3.6 is ended 9 months ago, but 3.7 is still supported for 9 month.

Reply:
Simply because 3.6 is still the default version on some older distributions, which are still supported (e.g. CentOS 7). That's why I kept it for now. But I didn't want to test against all old Python versions.